src/admin/python/cli/script.py

```diff
@@ -195,6 +196,10 @@ def smig_replication_controller(
     ----------
     connection : `str`
         Connection string in format mysql://user:pass@host:port/database
+    repl_connection : `str`, optional
+        The connection string for the replication manager database for the
+        non-admin user. Required when initializing the database, not needed
```

Comment:
This statement (regarding "non-admin") may be incorrect because this connection is used later by method `_create_database` to do the typical database administrator operations. These actions require administrative privileges in MySQL (or any other database):
```
            f"CREATE USER IF NOT EXISTS {user}@localhost;",
            f"CREATE USER IF NOT EXISTS {user}@'%';",
            f"CREATE DATABASE IF NOT EXISTS qservReplica;",
            f"GRANT ALL ON qservReplica.* TO  {user}@localhost;",
            f"GRANT ALL ON qservReplica.* TO  {user}@'%';",
            f"FLUSH PRIVILEGES;",
```

Reply:
Remind me, please, why do you have two connection strings in this app?
```
connection=
repl_connection=
```

Reply:
connection is the admin URI to the replication database. 

repl_connection is as the comment describes. The `connection` is used to create the user named in `repl_connection`

Reply:
Could it be also used as a source of the user's password? The password would be used in:
```
f"CREATE USER IF NOT EXISTS {user}@'%' IDENTIFIED BY '{password}';",
```
It's still fine to have the passwordless entries for logins from `localhost` as in:
```
f"CREATE USER IF NOT EXISTS {user}@localhost;",
```

Reply:
> Could it be also used as a source of the user's password?
yes, exactly. As discussed below. Will defer, I've added it to The List.